ls1intum/Artemis
# `Assessment`: Fix complaints not accessible after lock runs out

### Checklist
#### General


#### Client
#### Changes affecting Programming Exercises

### Motivation, Context and Description


Fix #5847

### Steps for Testing

Prerequisites:
- 3 Tutors
- 1 Student
- 1 Exercise with Complaints enabled

Best tested locally (set COMPLAINT_LOCK_DURATION_IN_MINUTES in Constants.java to 5 minutes)
1. Log in to Artemis
2. Make submission as student
3. Assess the submission as tutor 1
4. Complain as student
5. Lock the complaint as tutor 2
6. Wait until the lock should run out (normally 24h, with local change sooner)
7. Check that you can now assess the complaint as tutor 3

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
